packages/circuits/src/merkle.circom

```diff
@@ -13,7 +13,7 @@ template HashLeftRight() {
     signal output hash;
 
     // TODO: use more rounds
-    component hasher = MiMCSponge(2, 4, 1);
+    component hasher = MiMCSponge(2, 220, 1);
```

Comment:
Not really a concern for this, but did you have a look at proof generation times?